pinot-core/src/main/java/org/apache/pinot/core/plan/FilterPlanNode.java

```diff
@@ -70,9 +71,16 @@ public FilterPlanNode(IndexSegment indexSegment, QueryContext queryContext) {
     _numDocs = _indexSegment.getSegmentMetadata().getTotalDocs();
   }
 
+  public FilterPlanNode(IndexSegment indexSegment, QueryContext queryContext,
+      FilterContext filterContext) {
```

Comment:
Can we rename `filterContext` to `aggregationFilterContext` or something to indicate that this FilterPlanNode is constructed from AggregationPlanNode for FILTERed aggregation ?